src/main/java/com/jalasoft/jfc/controller/ImageConverterController.java

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2019 Jalasoft.
+ *
+ * This software is the confidential and proprietary information of Jalasoft.
+ *  ("Confidential Information"). You shall not disclose such Confidential
+ *  Information and shall use it only in accordance with the terms of the
+ *  license agreement you entered into with Jalasoft.
+ */
+
+package com.jalasoft.jfc.controller;
+
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.multipart.MultipartFile;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.io.IOException;
+
+/**
+ *
+ *  Manage ImageConvert Requests.
+ *
+ * @version 0.1.
+ *
+ * @author Enrique Carrizales.
+ */
+@RestController
+@RequestMapping(path = "/convertImage")
+public class ImageConverterController {
+
+    private static final String UPLOADED_FOLDER =
+            "src/main/java/com/jalasoft/jfc/resources/";  //Constant upload file
+
+    /**
+     * convertImage method receives a image to covert.
```

Comment:
Here correct **image to covert** there are misspellins **"an image to convert**"